### PR TITLE
use real navigator header for WyreWebview

### PR DESCRIPTION
**Pull request by brunobar79**
Looks like this: 
Screen Shot 2020-11-18 at 3 31 40 PM

### CODE REVIEW

**Comment by osdnk** [comment]
image

File: src/navigation/config.js [hunk 1]
```diff
@@ -233,6 +233,34 @@ const BackArrow = styled(Icon).attrs({
 `;
 const BackImage = () => <BackArrow />;
 
+export const wyreWebviewOptions = {
```
Comment (osdnk) [inline src/navigation/config.js:236]: have copied it? Maybe we can extract it?

**Review by osdnk — CHANGES_REQUESTED** [review]

**Review by osdnk — APPROVED** [review]
so cute

**Review by skylarbarrera — APPROVED** [review]
🍳

**Pull request merged**